cmd/ocitools/generate.go

```diff
@@ -58,6 +58,7 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "seccomp-errno", Usage: "specifies syscalls to be added to list that returns an error"},
 	cli.StringFlag{Name: "template", Usage: "base template to use for creating the configuration"},
 	cli.StringSliceFlag{Name: "label", Usage: "add annotations to the configuration e.g. key=value"},
+	cli.IntFlag{Name: "oom-score-adj", Usage: "specify oom_score_adj for the container"},
```

Comment:
@Mashimiao , I was struggling with this when I made the change.
The man doc always use `specifies`, `adds`, `uses`. However, most of flags defined in cmd/ocitools/generate.go are using `specify, `add`, `use`.

I will first change it to `a phrase` instead of `a verb + a noun`.